docs/root/configuration/cluster_manager/cluster_stats.rst

```diff
@@ -247,6 +247,7 @@ decisions. Stats are rooted at *cluster.<name>.* and contain the following stati
   lb_subsets_removed, Counter, Number of subsets removed due to no hosts
   lb_subsets_selected, Counter, Number of times any subset was selected for load balancing
   lb_subsets_fallback, Counter, Number of times the fallback policy was invoked
+  lb_subsets_fallback_panic, Counter, Number of times the fallback policy failed so any host was selected
```

Comment:
maybe "Number of times the subset panic mode triggered"?

Reply:
Done.